torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2012,6 +2012,100 @@ RegisterOperators reg2({
           return 0;
         }),
 
+    Operator(
```

Comment:
Looks good!! At this point, I think it would be good to add a macro for defining an op that is defined for both float and int. Maybe look at DEFINE_GENERIC_OP and do something similar ?

Reply:
I defined a macro, but I am not 100% confident on it

Reply:
Take a look, I subbed out my functions for the macro @eellison